openstudiocore/src/energyplus/Test/CoilWaterHeatingDesuperheater_GTest.cpp

```diff
@@ -57,7 +59,9 @@ TEST_F(EnergyPlusFixture, ForwardTranslator_CoilWaterHeatingDesuperheater) {
   std::vector<HVACComponent> testCoils = {
     CoilCoolingDXSingleSpeed(m),
     CoilCoolingDXTwoSpeed(m),
-    CoilCoolingDXTwoStageWithHumidityControlMode(m)
+    CoilCoolingDXTwoStageWithHumidityControlMode(m),
+    CoilCoolingDXVariableSpeed(m),
+    CoilCoolingDXMultiSpeed(m)
```

Comment:
ideally we should add a test for the other remaining objects that are non DX (I should have done that in the first place), I'll do that.

Reply:
Done in https://github.com/NREL/OpenStudio/pull/3565/commits/41a4de08b57878d3c798be2474cc6fba64d422ef